### PR TITLE
add rules and show error message if the target user of the warn is admin

### PR DESCRIPTION
**Pull request by iiiiii1wepfj**

### CODE REVIEW

File: plugins/rules.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@require_admin(permissions=["can_restrict_members"])
+async def delete_rules(c: Client, m: Message):
+    set_rules(m.chat.id, None)
+    await m.reply_text("the rules was deleted successfully.")
+ 
+        
+@Client.on_message(filters.command("rules", prefix) & filters.group)
+async def show_rules(c: Client, m: Message):
+    rules = get_rules(m.chat.id) or "the rules was not setted"
+    await m.reply_text(f"the rules in the chat {m.chat.title}: \n\n {rules}")
```
Comment (AndrielFR) [inline plugins/rules.py:40]: I think it would be better to send a button to the private to avoid Flood.

Comment (alissonlauffer) [reply plugins/rules.py:40]: Agree. Something like: "Click the button below to view chat rules."

Comment (iiiiii1wepfj) [reply plugins/rules.py:40]: idk

Comment (iiiiii1wepfj) [reply plugins/rules.py:40]: i think this is good in a group message

**Pull request merged**